### ISSUE TITLE
Fetch voices from AWS only once

### ISSUE THREAD

**Issue by delgod** [markcellis/X-ATC-Chatter#23]
It seems that each time I open the Simple ATC Panel, it fetches voices from AWS, which freezes XP12 for a bit.
It would be great to cache the first response and reuse it.
OR It would be great to fetch voices from AWS only when the `Voices` tab is activated.

**Comment by markcellis** [comment]
So I moved this so that it does this once when X-Plane starts and the change will be available in 1.7.4 beta 3.

That said, there is still a delay the very first time you open the ATC Panel as it needs to determine the closest airport based on your location. This is a bit of an arduous process and takes a bit of time. I'll add another issue to try and speed that up.
